src/data/pequiv.lean

```diff
@@ -5,6 +5,38 @@ Authors: Chris Hughes
 -/
 import data.set.lattice
 
+/-!
+
+# Partial Equivalences
+
+In this file, we define partial equivalences `pequiv`, which are a bijection between a subset of `α`
+and a subset of `β`. Notationally, a `pequiv` is denoted by "`≃.`" (note that the full stop is part
+of the notation). The way we store these internally is with two functions `f: α → option β` and the
+reverse function `g: β → option α`, with the condition that if `f a` is `option.some b`,
+then `g b` is `option.some a`.
+
+## Main results
+
+- `pequiv.of_set`: creates a `pequiv` from a set `s`,
+which sends an element to itself if it is in `s`.
```

Comment:
```suggestion
  which sends an element to itself if it is in `s`.
```
Etc. I think markdown wants wrapped bullets to be indented to match the first line.

Reply:
markdown will be the death of me :(